.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Expected Behavior
+<!--- If you're describing a bug, tell us what should happen -->
+<!--- If you're suggesting a change/improvement, tell us how it should work -->
+
+## Current Behavior
+<!--- If describing a bug, tell us what happens instead of the expected behavior -->
+<!--- Provide the full error output in a code block -->
+<!--- If suggesting a change/improvement, explain the difference from current behavior -->
+
+## Possible Solution
+<!--- Not obligatory, but suggest a fix/reason for the bug, -->
+<!--- or ideas how to implement the addition or change -->
+
+## Steps to Reproduce (for bugs)
+<!--- Provide the input image(s) and list the commands you entered. -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+<!--- Run `git rev-parse HEAD` in the ocropy directory -->
```

Comment:
I have never used the `git rev-parse` command so far. Maybe, it is easier to suggest `git log -n 1` where the user might easier see if the version is outdated.